Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -1179,6 +1179,7 @@ localizedStrings["Proportional Numerals @ Font Details Sidebar Property Value"]
 /* Property value for `font-variant-alternates: proportional-width`. */
 localizedStrings["Proportional-Width Variants @ Font Details Sidebar Property Value"] = "Proportional-Width Variants";
 localizedStrings["Protocol"] = "Protocol";
+localizedStrings["Pseudo"] = "Pseudo";
```

Comment:
Probably worth a localization description mentioning that this is part of a technical term from the spec somehow, and not just the English word "pseudo". Probably affects how it will get translated.